pkgs/development/tools/devcontainers-cli/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  nodejs_18
+}:
+let
+  pname = "devcontainers-cli";
+  version = "0.54.1";
+  hash = "sha256-L6sVDmKuFmrf9Bm9M54ABmmWFVB8ZlqU+5gAeITBS1Q=";
+in
```

Comment:
Why are these outside the `mkDerivation`?

Reply:
I don't mind putting `version` there, that makes sense. Alternatively finalAttrs. But I feel like it's a bit confusing to put `pname` and `hash` there as well, considering that both are only used once. It had me looking around for where else they were used.